src/GeositeFramework/js/Plugin.js

```diff
@@ -47,7 +47,43 @@ require(['use!Geosite',
                         downloadAsCsv: requestCsvDownload,
                         downloadAsPlainText: requestTextDownload,
                         dispatcher: N.app.dispatcher,
-                        suppressHelpOnStartup: _.partial(suppressHelpOnStartup, model)
+                        suppressHelpOnStartup: _.partial(suppressHelpOnStartup, model),
+                        resize: {
+                            adjustBy: function(val) {
+                                var currentWidth = parseFloat($uiContainer.css("width"));
```

Comment:
Not entirely consequential, but I don't think the `width` property can actually be a float, consider using `parseInt` instead.